cloudflare/resource_cloudflare_load_balancer_pool.go

```diff
@@ -119,6 +119,28 @@ var originsElem = &schema.Resource{
 			Optional: true,
 			Default:  true,
 		},
+
+		"header": {
+			Type:     schema.TypeSet,
+			Optional: true,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"header": {
```

Comment:
perhaps `name` instead of `header` to be more specific about the value intention?

```suggestion
					"name": {
```

Reply:
Hi @jacobbednarz, I'm not sure here because I see that naming in the cloudflare_load_balancer_monitor and cloudflare_healthcheck resources, and maybe will be better to use the existing one for consistency?

Reply:
I think it makes sense for those use cases however here, it's a little less helpful due to the outer block also being called `header` causing what appears like duplication but is actually intentional. Personally, it's quite confusing to see 

```hcl
// ...
header {
  header = "foo"
  values = ["bar"]
}
```

For the sake of not confusing people, it might be best to do something like

```hcl
// ...
header {
  name   = "foo"
  values = ["bar"]
}
```